src/server_manager/ui_components/app-root.html

```diff
@@ -497,6 +510,11 @@ <h4>Disconnect DigitalOcean account</h4>
         'SetUpGcpRequested': 'handleSetUpGcpRequested',
         'ManualServerEntryCancelled': 'handleManualCancelled'
       },
+      ready: function() {
+        // TODO(alalama): detect language
+        const messagesUrl = `/messages/${this.language}.json`;
+        this.loadResources(messagesUrl, this.language);
```

Comment:
Do we need this? Aren't we passing the localize function?

Reply:
We do - this is the function that loads the strings for `localize`.